refarch-backend/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# For Java applications use /ubi9/openjdk-21-runtime as base image, for documentation
+# please see https://jboss-container-images.github.io/openjdk/
```

Comment:
```suggestion
# For documentation see https://jboss-container-images.github.io/openjdk/
```
Isn't the first part not somehow a duplicate of just setting the image.
Think you can assume that the image was selected on purpose. What alternatively could be documented is why exactly this image was used.